src/lua/loaders.lua

```diff
@@ -330,4 +351,15 @@ return {
     -- It is `true` during tarantool initialization, but once all
     -- the built-in modules are ready, will be set to `nil`.
     initializing = true,
+    raw_require = raw_require,
```

Comment:
Hm, this made a bit inconsistent our structure with other modules overriding Lua stdlib functions. IDK, maybe all this should be moved to a separate `'internal.require'` module. Thoughts?

Reply:
`require()` is also wrapped in `src/lua/init.lua`. Sure, it worth to do. But this refactoring requires a bit more effort[^1], so I would pass it over for now. (This API is anyway internal and we can move this field to another internal module. I exposed `raw_require` just in case -- it should be accessible from somewhere IMHO, when we wrap it.)

[^1]: I have mentioned it in 432745263024641f45a0f0866a3c2691ef4b8a8b:

      > The init.lua file contains initialization code from different domains and it worth to split it further. I'll not do that in this series.

Reply:
OK, that works for me. Feel free to add this to your TODO list or file an issue.

Reply:
[Added](https://www.notion.so/tarantool/Move-require-wrappers-into-src-lua-require-lua-1bc8ae64107a420faeb7c56a5ff0d5cf).